app/src/main/java/org/gdg/frisbee/android/activity/MainActivity.java

```diff
@@ -364,11 +362,10 @@ public Chapter getSelectedChapter() {
             return mSelectedChapter;
         }
 
-        public void setSelectedChapter(Chapter chapter) {
+        public void setSelectedChapter(@NonNull Chapter chapter) {

```

Comment:
If you add `@NonNull` here than it's not worth checking the `null` again inside. You should safely assume that the client of the method is fulfilling the non null pre-condition. The IDE will tell the client of the method that a null argument should not be passed. So, either you remove `@NonNull` or remove the check for `!= null` inside.